src/instrumentsscene/view/instrumentspanelcontextmenumodel.cpp

```diff
@@ -64,6 +64,10 @@ void InstrumentsPanelContextMenuModel::load()
         m_masterNotation->parts()->scoreOrderChanged().onNotify(this, [this] {
             updateOrderingMenu(m_masterNotation->parts()->scoreOrder().id);
         });
+
+        m_masterNotation->parts()->partsChanged().onNotify(this, [this] {
```

Comment:
Could you please explain why you need this connection? scoreOrderChanged() is not enough?

Reply:
This is a leftover of earlier versions and indeed it isn't required anymore.
Removed.